benchmarking/src/cloudflare.ts

```diff
@@ -0,0 +1,95 @@
+import nodeFsPromises from "node:fs/promises";
+import nodeFs from "node:fs";
+import nodePath from "node:path";
+import nodeChildProcess from "node:child_process";
+import nodeUtil from "node:util";
+
+const promiseExec = nodeUtil.promisify(nodeChildProcess.exec);
+
+await ensureWranglerSetup();
+
+/**
+ * Collects name and absolute paths of apps (in this repository) that we want to benchmark
+ *
+ * @returns Array of objects containing the app's name and absolute path
+ */
+export async function collectAppPathsToBenchmark(): Promise<
+  {
+    name: string;
+    path: string;
+  }[]
+> {
+  const allExampleNames = await nodeFsPromises.readdir("../examples");
+
+  const examplesToIgnore = new Set(["vercel-commerce"]);
```

Comment:
Add a comment on why this is blocklisted

Reply:
done 🙂